buildscripts/02_toolchain

```diff
@@ -49,12 +42,11 @@ if ($C->HOST ne $C->TARGET) {
 
    # Install a standalone Android NDK toolchain.
    must_system (
-      $C->NDK_HOME . "/build/tools/make-standalone-toolchain.sh",
-      "--stl=$STL{$C->TARGET}",
-      "--ndk-dir=" . $C->NDK_HOME,
-      "--toolchain=$TOOLCHAIN{$C->TARGET}-clang",
-      "--install-dir=$toolchain",
-      "--platform=$PLATFORM{$C->TARGET}",
+      $C->NDK_HOME . "/build/tools/make_standalone_toolchain.py",
+      "--arch", $ARCH{$C->TARGET},
+      "--install-dir", $toolchain,
+      "--api", $API{$C->TARGET},
+      "--force",
```

Comment:
Just to note: I used `--force` in Jenkins scripts because without it `make_standalone_toolchain.py` won't put a toolchain into an existing `--install-dir` directory, even if it's empty.